articles/active-directory/devices/howto-device-identity-virtual-desktop-infrastructure.md

```diff
@@ -60,7 +60,7 @@ Before configuring device identities in Azure AD for your VDI environment, famil
 | Azure AD registered | Federated/Managed | Windows current/Windows down-level | Persistent/Non-Persistent | Not Applicable |
 
 <sup>1</sup> **Windows current** devices represent Windows 10, Windows Server 2016, and Windows Server 2019.
```

Comment:
Then just say "Windows Server 2016 v1803 or higher" here on line 62. The note doesn't make sense and people wont read it.

Reply:
That's what my initial suggestion to update the doc to Ravi Venapusa. But he replied below why we put windows Server 2016 as supported instead of mentioning current version Windows server 2016 v1803
================
In general, we support persistent VDI on WS2016 to the extent of what’s available in v1607. But there may be new features or improvements that do not apply to that version. So customers either have to patch those servers with a latest update or migrate to a future version to get the benefit of those improvements.  
=============